decidim-core/app/views/decidim/account/_account_notification.js.erb

```diff
@@ -0,0 +1,15 @@
+if (typeof accountNotification === 'undefined') {
```

Comment:
I think it's generally safer (and cleaner) to always define the variable in these types of cases, no matter if it's already defined before.

Same also below for `messageDiv`.